packages/components/multiselect/src/Multiselect.tsx

```diff
@@ -0,0 +1,278 @@
+import React, { useRef, useState } from 'react';
+import { cx } from 'emotion';
+
+import { mergeRefs, type CommonProps } from '@contentful/f36-core';
+import { Button, IconButton } from '@contentful/f36-button';
+import { TextInput, type TextInputProps } from '@contentful/f36-forms';
+import { CloseIcon, ChevronDownIcon, SearchIcon } from '@contentful/f36-icons';
+import { SkeletonContainer, SkeletonBodyText } from '@contentful/f36-skeleton';
+import { Popover, type PopoverProps } from '@contentful/f36-popover';
+import { Subheading } from '@contentful/f36-typography';
+
+import { getMultiselectStyles } from './Multiselect.styles';
+
+export interface MultiselectProps
+  extends CommonProps,
+    Pick<
+      TextInputProps,
+      | 'isDisabled'
+      | 'isInvalid'
+      | 'isReadOnly'
+      | 'isRequired'
+      | 'id'
+      | 'defaultValue'
+    > {
+  /** Select Options */
+  children?: React.ReactNode;
+
+  /**
+   * Set a custom icon for the text input
+   */
+  startIcon?: React.ReactElement;
+
+  /**
+   * Placeholder shown before selecting any elements. Defaults to 'Select one or more items'
+   */
+  placeholder?: string;
+
+  /**
+   * current Selected items, to be shown on the trigger button
+   */
+  currentSelection?: Array<string>;
+
+  /**
+   * If this is set to `true` there will be a search input inside the drawer
+   * @default false
+   */
+  hasSearch?: boolean;
```

Comment:
Isn't this implicitly defined by passing `onSearchValueChange`? Or is there a case where you define only one of them?

Reply:
that is actually a very fair point, saves us one prop and makes the implementation more straigth forward!! 🚀